docs/source/tracking.rst

```diff
@@ -370,6 +370,36 @@ This path must the same on both the server and the client -- you may need to use
 the client in order to enforce this property.
 
 
+HDFS
+^^^^^
+
+To store artifacts in an HDFS, specify a URI of the form ``hdfs:/<path>``. It can contain host and port:
+
+|     ``hdfs:/localhost:8020/path`` or just the path:
+|     ``hdfs:/<path>``
+|
+
+Choice is dependant on the authorization in use. Supporter are two cases:
+
+- use current account authorization (Linux account)
+- Kerberos credentials using following environment variables:
+
+.. code-block:: bash
+
+  export MLFLOW_KERBEROS_TICKET_CACHE=/tmp/krb5cc_22222222
+  export MLFLOW_KERBEROS_USER=user_name_to_use
+
+Most of the cluster contest settings is read from hdfs-site.xml accessed by the hdfs native driver
```

Comment:
Nit: put `hdfs-site.xml` in backtracks so that it shows up as code; also capitalize HDFS